tests/benchdnn/inputs/graph/pattern/f8/f8_bf16_matmul_add_fusion.json

```diff
@@ -0,0 +1,365 @@
+{
+    "version": "3.0.0",
```

Comment:
please update the json with the latest version

Reply:
Thanks for reminding. Have fixed.